apps/deriv-hk/pages/index.tsx

```diff
@@ -6,6 +6,11 @@ export function Index() {
   return (
     <>
       <Head>
+        <meta
+          name="viewport"
+          key="viewport"
+          content="width=device-width, initial-scale=1.0"
+        />
```

Comment:
We need this in all pages yeah?

Reply:
needed in homepage, help center and trade page. should we customize this?

Reply:
can you please move this to `_document.tsx`?